### DISCUSSION TITLE
nodemon working but changes are not reflected

### DISCUSSION THREAD

**Discussion by janbaztaimoor** [lerna/lerna#3631] [Q&A]
I am learning Node from a YouTube tutorial and got stuck on nodemon problem...

OS Ubuntu 22.04, IDE VSCode

here is my package.json

{
  "name": "devops_with_node_n_express",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "start": "node index.js",
    "dev":  "nodemon -L index.js"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "dependencies": {
    "express": "^4.18.2"
  },
  "devDependencies": {
    "nodemon": "^2.0.22"
  }
}

result of issuing nodemon command

ubuntuserver:~/Documents/2-Dockers_and_Containers_Resources/DevOps_with_Node_n_Express$ `nodemon index.js`
[nodemon] 2.0.22
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): *.*
[nodemon] watching extensions: js,mjs,json
[nodemon] starting `node index.js`
listening on port 3600
[nodemon] restarting due to changes...
[nodemon] starting `node index.js`
listening on port 3600

Help will be deeply appreciated

**Comment by fahslaj** [comment]
Hi @janbaztaimoor, I am sorry that you are stuck. Could you please elaborate on your specific problem, what you are expecting to happen, and how it relates to Lerna?

**Comment by janbaztaimoor** [comment]
I am sorry. I Think I posted the question at wrong place.
Apologies for inconvenience.
